src/resources/CaseAssistConfigs/CaseAssistPreviewInterfaces.ts

```diff
@@ -0,0 +1,39 @@
+import {CaseAssistConfigModel} from './CaseAssistConfigInterfaces';
+
+export interface Document {
+    title: string;
+    clickUri: string;
+    excerpt: string;
+    fields: any;
+}
+
+export interface DocumentSuggestions {
+    documents: Document[];
+    totalCount: number;
+}
+
+export interface FieldValueSuggestion {
+    predictions: Array<{
+        value: string;
+        confidence: number;
+    }>;
+}
+
+export interface CaseClassifications {
+    fields: Record<string, FieldValueSuggestion>;
+}
+
+export interface ContextField {
+    value: string;
+}
+
+export interface ContextFields {
+    [key: string]: ContextField;
+}
+
+export interface PreviewRequestBody {
+    visitorId: string;
+    locale: string;
+    fields: ContextFields;
+    configuration: Partial<CaseAssistConfigModel>;
```

Comment:
Are you sure about attempting to get a preview from an incomplete config? I don't think we should allow that.